src/diffusers/loaders/lora_pipeline.py

```diff
@@ -197,7 +198,8 @@ def load_lora_weights(
             pretrained_model_name_or_path_or_dict = pretrained_model_name_or_path_or_dict.copy()
 
         # First, ensure that the checkpoint is a compatible one and can be successfully loaded.
-        state_dict, network_alphas = self.lora_state_dict(pretrained_model_name_or_path_or_dict, **kwargs)
+        kwargs["return_lora_metadata"] = True
```

Comment:
I think this is okay to do otherwise we end up introducing another argument to `load_lora_weights()`.